client/client.go

```diff
@@ -211,6 +212,11 @@ func WithBuckets() GetRegionOption {
 	return func(op *GetRegionOp) { op.needBuckets = true }
 }
 
+// AllowFollowerHandle means that client can send request to follower and let it handle this request.
+func AllowFollowerHandle() GetRegionOption {
```

Comment:
How about keeping consistent with the HTTP interface and changing it to WithAllowFollowerHandle?

Reply:
gRPC uses lower case, but HTTP uses MIME format. So I think no need to keep consistent.